lib/topics.js

```diff
@@ -8,18 +8,37 @@ const SNS    = require('./sns');
 class Topic {
 
   constructor(server, topicName) {
-    this._server = server;
-    this.name    = topicName;
+    this._server     = server;
+    this.name        = topicName;
+    this.subscribers = [];
```

Comment:
`Set`? It would only a subscribed to be added once, and just as easily removed. Unlike queues which are persistent, subscriptions can be ephemeral, you can add/remove freely at runtime.